_guidelines-v4/04-cmn/03-cmnadditionalmeasurecontent/04-cmntuplets.md

```diff
@@ -21,4 +21,4 @@ In addition to {% include link elem="note" %} elements, {% include link elem="tu
 The {% include link elem="tuplet" %} element may also be used for repetition of the same pitch; that is, a single note or chord may be the only content of the tuplet. In some cases, optical music recognition software may treat these instances as bowed tremolandi due to the knowledge of the complete semantics of the notation at the time of recognition. However, marking these as tuplets is the recommended practice.
 
 In some situations, a tuplet is made up of events in different measures. As this raises the issue of non-concurrent hierarchies, it is not possible to encode such situations with the {% include link elem="tuplet" %} element described above. Therefore, MEI offers the {% include link elem="tupletSpan" %} element, which is member of the {% include link model="controlEventLike" %} class. It is nested inside of {% include link elem="measure" %}, following all the measure's {% include link elem="staff" %} children. It uses the same attributes as {% include link elem="tuplet" %} to describe tuplets, but instead of nesting all affected notes inside itself, it references the **@xml:id** values of all affected notes in its **@plist** attribute and the initial and terminal notes of the tuplet using **@startid** and **@endid** attributes. This configuration allows tuplets to cross measure boundaries. The following example demonstrates a typical example of such hierarchy-crossing tuplets:
-{% include mei example="cmn/cmn-sample141.xml" valid="" %}
+{% include mei example="cmn/cmn-sample141.txt" valid="" %}
```

Comment:
this example should be improved, in that it should have `@num` and `@numbase` attributes for complete semantics. It would also be more comprehensible if it had just three notes referenced in its `@plist` attribute (although an image of such a `<tupletSpan>` would be even better, that's not a must)